src/pkg/cli/compose/validation.go

```diff
@@ -206,23 +208,23 @@ func ValidateProject(project *composeTypes.Project) error {
 				}
 			}
 		} else {
-			timeout := 30 // default per compose spec
+			timeout := 5.0
```

Comment:
This change allows you to finally add `interval: 10s` to the `healthcheck:` in Compose to trigger the debugger faster.